tests/smoke.rs

```diff
@@ -115,3 +115,12 @@ fn with_seed() {
     b.seed(7);
     assert_eq!(a.u64(..), b.u64(..));
 }
+
+#[test]
+fn choose_multiple() {
+    let a = fastrand::Rng::with_seed(7);
+    assert_eq!(
+        a.choose_multiple(0..20, 12),
+        vec![0, 1, 17, 15, 4, 14, 6, 7, 19, 9, 18, 13]
```

Comment:
Would it be possible to avoid hardcoding the results here? If we move to a different algorithm, then this test will fail.